synctl

```diff
@@ -307,12 +334,21 @@ def main():
             for cache_name, factor in iteritems(worker.cache_factors):
                 os.environ["SYNAPSE_CACHE_FACTOR_" + cache_name.upper()] = str(factor)
 
-            start_worker(worker.app, configfile, worker.configfile)
+                errors += int(start_worker(worker.app, configfile, worker.configfile))
```

Comment:
likewise, this is now part of the cache_factors loop, and the int cast is wrong (I think)